source/read/project.txt

```diff
@@ -0,0 +1,137 @@
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document supplied to a 
+query that specifies which fields MongoDB returns.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<pymongo-get-started>` guide.
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. With the exception of the
+``_id`` field, you can't combine inclusion and exclusion statements in
+a single projection.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you use a projection to specify fields to include, any unspecified fields
+are implicitly excluded from the return document. 
+
+.. note::
+    
+   The ``_id`` field is *not* implicitly excluded from the return document when
+   it isn't included in the projection. To remove the ``_id`` field from the return
+   document, you must :ref:`explicitly exclude it <pymongo-project-remove-id>`.
+
+Specify fields to include with the following syntax:
+
+.. code-block:: json
+
+   { "<Field Name>": 1 }
+
+The following example uses the ``find()`` method to find all restaurants with
+the ``name`` field value of ``"Emerald Pub"``. It then uses a projection to
+return only the ``name``, ``cuisine``, and ``borough`` fields in the returned
+documents:
+
+.. io-code-block::
+    
+   .. input:: /includes/project/project.py
+      :start-after: start-project-include
+      :end-before: end-project-include
+      :language: python
+      :emphasize-lines: 1
+
+   .. output::
+
+      {'_id': ObjectId('...'), 'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub'}
+      {'_id': ObjectId('...'), 'borough': 'Queens', 'cuisine': 'American', 'name': 'Emerald Pub'}
+
+.. _pymongo-project-remove-id:
+
+Exclude the ``_id`` Field
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When specifying fields to include, you can also exclude the ``_id`` field from
+the returned document.
+
+The following example performs the same query as the preceding example, but
+excludes the ``_id`` field in the projection:
+
+.. io-code-block::
+    
+   .. input:: /includes/project/project.py
+      :start-after: start-project-include-without-id
+      :end-before: end-project-include-without-id
+      :language: python
+      :emphasize-lines: 1
+
+   .. output::
+
+      {'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub'}
+      {'borough': 'Queens', 'cuisine': 'American', 'name': 'Emerald Pub'}
+
+Specify Fields to Exclude
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you use a projection to specify which fields to exclude,
+any unspecified fields are implicitly included in the return document.
```

Comment:
S: Can arguably leave this out, since you mentioned it in the Include section

Reply:
I think since it's relatively short it might be good to have this in case someone just jumps to that section